palantirnet/search_api_federated_solr
# Test federated-search-react 3.0.2

## Do not merge this PR. It is only meant to be checked out to test the React app release before pushing to the CDN.

## Testing

- Setup the Federated Search Demo.
- Go to `src/search_api_federated_solr` in the demo project.
- Checkout this branch.
- `drush cr`
- Visit http://d8.fs-demo.local/search-app
- Verify the app works as intended.
